test/ComposerRequireCheckerTest/DefinedSymbolsLocator/LocateDefinedSymbolsFromASTRootsTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace ComposerRequireCheckerTest\DefinedSymbolsLocator;
+
+use ArrayObject;
+use ComposerRequireChecker\DefinedSymbolsLocator\LocateDefinedSymbolsFromASTRoots;
+use PhpParser\Node\Stmt\Class_;
+use PHPUnit\Framework\TestCase;
+
+class LocateDefinedSymbolsFromASTRootsTest extends TestCase
```

Comment:
Please add `/** @covers \ComposerRequireChecker\DefinedSymbolsLocator\LocateDefinedSymbolsFromASTRoots */` to prevent coverage leaks